enigma-core/enclave/Cargo.toml

```diff
@@ -25,18 +25,17 @@ etcommon-hexutil = { version = "0.2", default-features = false }
 etcommon-bigint = { version = "0.2", default-features = false, features = ["rlp"] }
 rustc-hex = { version = "2.0", default-features = false }
 
-pwasm-utils = { git = "https://github.com/enigmampc/wasm-utils.git", rev = "sgx-1.0.7", default-features = false }
```

Comment:
This isn't `0.2.0` https://github.com/enigmampc/wasm-utils/tree/0.2.0-sgx-1.0.8
more like `0.5.0` https://github.com/enigmampc/wasm-utils/blob/0.2.0-sgx-1.0.8/Cargo.toml#L3